.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
I think we should continue to allow blank issues - it's reasonably buried, and it's annoying when none of the templates quite fits.